IMC_WebDev/#Driving_the_car_and_flowing_the_water_with_CSS/blog.md

```diff
@@ -1,16 +1,17 @@
 # Introduction to Web Animations using CSS
 
-In this blog, we are going to keep building our mini city. We will have a car going along the road and the bridge, and a boat going under the bridge.
+In this blog, we are going to keep building our mini city. We will have a car going along the road and the bridge as well as a boat going under the bridge.
```

Comment:
It'd be helpful to add a few more sentences prepping the reader for this blog, like adding something about how this post is about web animations using CSS. (This will help tie the intro into the conclusion too.)